src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,121 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import './NewMovie.scss';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  createMovie = (event) => {
```

Comment:
this func doesn't create a movie, it's handle input's event and write values to the state

```suggestion
  handleChange = (event) => {
```